src/test/java/com/wikia/webdriver/testcases/mercurytests/LightboxTests.java

```diff
@@ -112,11 +114,13 @@ public void MercuryLightboxTest_003_ZoomByGesture_ZoomByDoubleTap() {
     touchAction.zoomInOutPointXY(50, 50, 50, 100, PerformTouchAction.ZOOM_WAY_IN, 3000);
     File afterZooming = new Shooter().capturePage(driver);
```

Comment:
For that test we don't need to get such detailed comparison results.
For that tests steps are:
- capture image 1
- zoom in
- capture image 2
- compare 1 and 2, if they are different mark zoom in as pass
- zoom out
- capture image 3
- compare 1 and 3, if they are the same mark zoom out as pass

Byte difference for that test is nearly doubled:
- zoom out about 250kb
- zoom in about 500kb

So checking byte lenght is fine for that